src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -232,10 +236,11 @@ export class AccountManagementService implements IAccountManagementService {
 	}
 
 	/**
-	 * Retrieves all the accounts registered with ADS.
+	 * Retrieves all the accounts registered with ADS based on auth library in use.
 	 */
 	public getAccounts(): Promise<azdata.Account[]> {
-		return this._accountStore.getAllAccounts();
+		return this._accountStore.getAllAccounts()
```

Comment:
Would be preferable to just make this async and await the getAllAccounts call

Reply:
Opened issue https://github.com/microsoft/azuredatastudio/issues/21502 to track this.